include/proxy/v4/proxy.h

```diff
@@ -70,15 +74,19 @@ struct reduction_traits {
 };
 
 template <class Expr>
-consteval bool is_consteval(Expr)
-    { return requires { typename std::bool_constant<(Expr{}(), false)>; }; }
+consteval bool is_consteval(Expr) {
+  return requires { typename std::bool_constant<(Expr{}(), false)>; };
+}
 
 template <class T, std::size_t I>
 concept has_tuple_element = requires { typename std::tuple_element_t<I, T>; };
 template <class T>
 consteval bool is_tuple_like_well_formed() {
-  if constexpr (requires { { std::tuple_size<T>::value } ->
-      std::same_as<const std::size_t&>; }) {
+  if constexpr (requires {
```

Comment:
Can we fine tune the styles for this part?

Reply:
I couldn't find a better rule for this